ml-algorithms/build.gradle

```diff
@@ -10,6 +10,7 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
Is there something required to pull from the local or just make the process faster with local?

Reply:
Ignore this part, this is for debugging the tribuo bug https://github.com/oracle/tribuo/pull/224. Some tests will fail. Will ignore the flaky part first to pass CI.